mobile/src/components/Header.js

```diff
@@ -1,9 +1,15 @@
 import React, { Component } from 'react';
 import { Text, View, TouchableOpacity, Dimensions } from 'react-native';
 import Icon from 'react-native-vector-icons/FontAwesome';
+import MaterialIcon from 'react-native-vector-icons/MaterialCommunityIcons';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import * as Animatable from 'react-native-animatable';
+
 import commonStyles from '../styles/commonStyles';
 
+const AnimatableMaterialIcon = Animatable.createAnimatableComponent(MaterialIcon);
```

Comment:
I am not sure if you need this library. I think you could achieve this with https://facebook.github.io/react-native/docs/animated (but i guess you can leave it for now)